python/ray/_raylet.pyx

```diff
@@ -667,6 +667,7 @@ cdef class CoreWorker:
         options.raylet_socket = raylet_socket.encode("ascii")
         options.job_id = job_id.native()
         options.gcs_options = gcs_options.native()[0]
+        options.enable_ray_log = True
```

Comment:
Can we just name this "enable_logging"? Not sure why it needs to be called ray log

Reply:
Sure. Updated.